### PR TITLE
Password update cb

### PR DESCRIPTION
**Pull request by HigginsCB**
Updated to show guidelines for password and show active password strength/suggestions.  The guidelines show a basis of how to create a good password with zxcvbn. The active password strength meter updates based on zxcvbn score accompanied by the zxcvbn crack time and feedback for suggestions and warnings.

### CODE REVIEW

File: src/components/RegisterTab/RegisterTab.tsx [hunk 1]
```diff
@@ -95,9 +99,15 @@ export class RegisterTab extends Component<RegisterTabProps, {personalEmailTouch
           Please type in format of XXX-XXX-XXXX
         </Form.Control.Feedback>
       </Form.Group>
-
       <Form.Group controlId='formBasicPassword'>
-        <Form.Label column={true}>Password</Form.Label>
+        <Form.Label column={true}>New Password</Form.Label>
+        <p style={pStyle}>
```
Comment (jasekiw) [inline src/components/RegisterTab/RegisterTab.tsx]: I would put this below the password box.  Use bootstrap's example to add text below the input: https://react-bootstrap.github.io/components/forms/#forms

Why did you rename the password to "New Password"? This is the registration, not the change password page. You will need to do this on the change password page as well.

Change the text to:
```
We require a strong password for using our system. To increase the password strength, add numbers, special character, and increase password length.
```

File: src/components/AccountManagementLayout/PasswordStrength.tsx [hunk 2]
```diff
@@ -7,20 +7,32 @@ interface PasswordStrengthProps {
 }
 export default class PasswordStrength extends Component<PasswordStrengthProps> {
   createPasswordLabel(result: ZXCVBNResult) {
+    let x;
```
Comment (jasekiw) [inline src/components/AccountManagementLayout/PasswordStrength.tsx]: I don't think it is necessary to say what the crack time is. After looking at my other comment, I would revert these changes as saying "Strong" should be plenty since the description says you need a "Strong" password.

**Review by jasekiw — CHANGES_REQUESTED** [review]

File: src/components/RegisterTab/RegisterTab.tsx (hunk 1 above)
Comment (HigginsCB) [reply src/components/RegisterTab/RegisterTab.tsx]: I was copying and pasting from both the registration page and the reset password page and I think I missed that when I pasted.

File: src/components/AccountManagementLayout/PasswordStrength.tsx (hunk 2 above)
Comment (HigginsCB) [reply src/components/AccountManagementLayout/PasswordStrength.tsx]: So just have the old meter and have the password guidelines or do you want to have it show the suggestions still just not the warnings?

**Comment by HigginsCB** [comment]
Fixed requested issues with password guidelines and suggestions

File: src/components/RegisterTab/RegisterTab.tsx [hunk 3]
```diff
@@ -107,8 +110,11 @@ export class RegisterTab extends Component<RegisterTabProps, {personalEmailTouch
           isInvalid={!isPassValid(this.props.form.password)}
           placeholder='Password'
         />
-      </Form.Group>
-      <PasswordStrength password={this.props.form.password}/>
+        <Form.Text className='text-muted'>
+          We require a strong password for using our system.
```
Comment (jasekiw) [inline src/components/RegisterTab/RegisterTab.tsx]: Instead of duplicating the text and style in two different places, why don't you create a component to hold the text and use the component both in the register tab and the reset password page.

**Review by jasekiw — CHANGES_REQUESTED** [review]

**Review by jasekiw — APPROVED** [review]
